internal/integration/spanner_test.go

```diff
@@ -30,7 +30,7 @@ type spannerTest struct {
 }
 
 var spannerTests = map[string]*spannerTest{
-	"spanner-emulator": {port: 9020},
+	"spanner-emulator": {port: 9010},
```

Comment:
unrelated but this reflects the gRPC port that is used for the spanner emulator traffic.